tests/test_database.py

```diff
@@ -42,3 +42,16 @@ class TestDatabase(tb.ConnectedTestCase):
             await conn.aclose()
         finally:
             await self.con.execute('DROP DATABASE mytestdb;')
+
+    async def test_database_current_database_01(self):
+        await self.con.execute('CREATE DATABASE currentdb;')
```

Comment:
Wait, this is a bit too slow. Just add this test to test_database_create_01

Reply:
Errr, sorry, I just merged. I can make a quick PR with tests consolidation.